src/app/utils/validateString.js

```diff
@@ -0,0 +1,13 @@
+export default function validateString(text) {
+  const isString = typeof text === 'string';
+  if (!isString) {
+    throw TypeError('Invalid data');
+  }
+
+  const hasCharacters = text.trim().length > 0;
+  if (!hasCharacters) {
+    throw new Error('Text is Empty');
+  }
```

Comment:
!isString || !hasCharacters ? throw : 1

Reply:
Very short indeed :)

Reply:
🔝 I forgot about the brackets (it's an expression)
@vieraboschkova

Reply:
I came across this: https://stackoverflow.com/questions/9370606/javascript-error-handling-can-i-throw-an-error-inside-a-ternary-operator and perhaps you could relate to that? I mean if throwing errors using ternary expressions is a right approach

Reply:
You're absolutely right. I didn't realise it has happened there